replicate-to-sql-with-generic-events/scripts/delete-data.js

```diff
@@ -0,0 +1,47 @@
+const { Kuzzle, WebSocket } = require('kuzzle-sdk');
+
+const kuzzle = new Kuzzle(new WebSocket('localhost'));
+
+const indexName = 'nyc-open-data';
+const collectionName = 'yellow-taxi';
+
+async function searchData() {
+  try {
+    const result = await kuzzle.document.search(
+      indexName,
+      collectionName,
+      {},
+      { size: 500 }
+    );
```

Comment:
Nitpicking: wrong coding standard, the last parenthesis should be on the last line (for saving an unnecessary line)